CIP-XXXX/README.md

```diff
@@ -0,0 +1,1597 @@
+    "properties": {
+      "cc_members": {
+        "type": "array",
+        "items": {
+          "$ref": "#/cip-139/CCMember"
+        }
+      }
+    },
+    "required": [
+      "cc_members"
+    ]
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get all known committee members.
+
+###### Id
+
+```
+{
+  "operation": "getCommitteeMemberById",
+  "request": {
+    "$ref": "#/cip-139/CCHotId"
+  },
+  "response": {
+    "$ref": "#/cip-139/CCMember"
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get a specific Committee member by id.
+
+##### Pool
+
+###### All
+
+```
+{
+  "operation": "getAllPools",
+  "request": {},
+  "response": {
+    "type": "object",
+    "properties": {
+      "pools": {
+        "type": "array",
+        "items": {
+          "$ref": "#/cip-139/Pool"
+        }
+      }
+    },
+    "required": [
+      "pools"
+    ]
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get all known stake pools.
+
+###### Id
+
+```
+{
+  "operation": "getPoolById",
+  "request": {
+    "$ref": "#/cip-139/PoolPubKeyHash"
+  },
+  "response": {
+    "$ref": "#/cip-139/Pool"
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get a specific stake pool by id.
+
+##### Proposal
+
+###### All
+
+```
+{
+  "operation": "getAllProposals",
+  "request": {},
+  "response": {
+    "type": "object",
+    "properties": {
+      "proposals": {
+        "type": "array",
+        "items": {
+          "$ref": "#/cip-139/Proposal"
+        }
+      }
+    },
+    "required": [
+      "proposals"
+    ]
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get all known proposals.
+
+###### Id
+
+```
+{
+  "operation": "getProposalById",
+  "request": {
+    "$ref": "#/cip-139/ProposalId"
+  },
+  "response": {
+    "$ref": "#/cip-139/Proposal"
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get a specific proposal by id.
+
+##### Era
+
+###### Summary
+
+```
+{
+  "operation": "getEraBySummary",
+  "request": {},
+  "response": {
+    "type": "object",
+    "properties": {
+      "summary": {
+        "type": "array",
+        "items": {
+          "$ref": "#/cip-139/EraSummary"
+        }
+      }
+    },
+    "required": [
+      "summary"
+    ]
+  },
+  "errors": [
+    {
+      "$ref": "#/appendix/APIError"
+    }
+  ]
+}
+```
+
+Get the start and end of each era along with parameters that can vary between hard forks.
+
+
+### Versioning
+
+In this CIP we are defining two different APIs: the connection API for wallets, and the CIP-30 *[maybe this should have another name to prevent confusion?]* extension which enables an own-data wallet. These two are separate components, at the top of this document there is a table with separate entries for the versions of the connection API and CIP-30 Extension respectively. 
```

Comment:
> *[maybe this should have another name to prevent confusion?]*

Good question. Let's leave this as an open question and see what the CIP editors recommend.